### ISSUE TITLE
Logging not working

### ISSUE THREAD

**Issue by Fabian1976** [deric/puppet-zookeeper#108]
The logfile of Zookeeper is not being written.

In my /var/log/messages i get this:
```
Oct 17 13:24:29 puppetdev-fabian01 java: java.io.FileNotFoundException: ./zookeeper.log (Permission denied)
```

In the log4j.properties, the ```zookeeper.log.dir``` is set to ```.```
So ```./zookeeper.log``` is legit.

In ```/etc/zookeeper/java.env``` this is set as the logdir: ```ZOO_LOG_DIR=/var/log/zookeeper``` but zookeeper seems to ignore this.

When I change the log4j template on my puppetmaster from this:
```
zookeeper.log.dir=.
```
to this
```
zookeeper.log.dir=<%= scope.lookupvar("zookeeper::log_dir") %>
```

It works fine.

I use a Satellite with zookeeper 3.4.10 compiled from source using this spec file: [spec-file](https://github.com/id/zookeeper-el7-rpm)

**Comment by Fabian1976** [comment]
I found the problem. When the default service_file is used, the log4j.properties file is used (as it should do).

When I set this parameter in my hiera yaml:
```
zookeeper::manage_service_file: true
```

The service file is managed by this module and all king of parameters are added as a startup parameter which are also in log4j.properties.

But then is does work.

I still think the log location should be properly set in log4j.properties. Why else use that file if you're going to override the ```zookeeper.log.dir``` parameter by adding it to the startup parameter. I believe conf files are there so that there is no need for adding everything as a startup parameter.
